DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/EmailDelegates/Index.cshtml

```diff
@@ -25,32 +25,44 @@
       <vc:error-summary order-of-property-names="@(new[] { nameof(Model.SelectedDelegateIds), nameof(Model.Day), nameof(Model.Month), nameof(Model.Year) })" />
     }
 
-    <span class="nhsuk-body-l">Select delegates to send email to.</span>
+    <span class="nhsuk-body-l">Filter and select delegates to send emails to.</span>
+
+    <input type="hidden" id="filter-by" name="filterBy" value="@Model.FilterBy" />
+
+    <div class="nhsuk-grid-row">
+      <div class="nhsuk-grid-column-full">
+        <partial name="SearchablePage/_Filters" model="Model" />
+      </div>
+    </div>
+
+    <div class="nhsuk-grid-row">

```

Comment:
You need to get latest master and use the new partials, otherwise this will break. You'll end up with duplicate elements with id filter-by and duplicate current filters.